bootstrap/absolute.sh

```diff
@@ -17,7 +17,10 @@ set_path_env $(bootstrap_command_path)
 set_path_env $(absolute_path)/node_modules/.bin
 
 # NPM update
-npm update
+if [ ! .pkg_timestamp -nt package.json ]; then
+  npm update
```

Comment:
I think we should check whether |npm update| is success or not.
npm update && touch .pkg_timestamp